pkg/controllers/data.go

```diff
@@ -0,0 +1,141 @@
+package controllers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/interactor"
+	"github.com/gorilla/mux"
+)
+
+type DataController struct {
+	DataInteractor interactor.DataInteractor
+}
+
+type errorMessage struct {
+	error   bool
+	message string
+}
+
+func NewDataController(dataInteractor *interactor.DataInteractor) *DataController {
+	return &DataController{*dataInteractor}
+}
```

Comment:
Sames as the previous comment :grimacing: 

```suggestion
type DataController struct {
	DataInteractor *interactor.DataInteractor
}

type errorMessage struct {
	error   bool
	message string
}

func NewDataController(dataInteractor *interactor.DataInteractor) *DataController {
	return &DataController{dataInteractor}
}
```

Reply:
Done!